website/static/worker.js

```diff
@@ -1,8 +1,10 @@
-/* eslint-env worker */
-/* eslint no-var: off, strict: off */
 /* globals prettier prettierPlugins parsersLocation */
 
-var imported = Object.create(null);
+"use strict";
```

Comment:
We have `parser-flow.js` which is not `strict`, and this test fails when I try to set it `strict` https://github.com/prettier/prettier/runs/676631327#step:7:65, but [this code](https://deploy-preview-8923--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEBnADgQzHVACAXjwB4ATASwDcA+AXTwYG1iB6CmkAGhAnRnOipkoTACdREAO4AFMQiEpMAG0mYAnkO4AjUdgDWcGAGUsYclADmyGKICucbnAC2WuKVJuAMpku3MFuAAxCFEnTBh+S2QQTFsYCC4QAAsYJyUAdSTyeAxsOCN5bKpstWiwVE0Qc1Q4URhpXQsw5AAzZRruACtUAA8AIV0wA2NMJzhPczhW9ocQbp6jcwslOABFWwh4aaUOkCxRGtFolqUpRPRRcxh08lIYJOQADgAGbguIGvTddGiL3FrKFNuABHDbwBq8BQxVAAWigcDcbkSojgoPIKIa-maSDaO1mNSc5GsdnxSxW602UxxM24MEwWhudweSAATLTdOQlEsAMIQJzYkC4ACsiVsNQAKvSFLjdpR7ABJKAeWBGMCXPgAQSVRhgahW2xqAF9DUA) seems working on background.


So it should not breaking things.

Reply:
> If you put "use strict" at the top of a web worker javascript file, it will not cause strict mode to be applied to any other files, including those imported via importScripts

Quote from https://stackoverflow.com/questions/26855884/is-the-function-form-of-use-strict-necessary-in-a-web-worker